packages/block-library/src/navigation/index.php

```diff
@@ -71,6 +71,18 @@ function block_core_navigation_sort_menu_items_by_parent_id( $menu_items ) {
  * Helper functions used to render the navigation block.
  */
 class WP_Navigation_Block {
+	/**
+	 * Returns whether or not this is responsive navigation.
+	 */
+	private static function is_responsive_navigation( $attributes ) {
```

Comment:
```suggestion
	private static function is_responsive( $attributes ) {
```

Do we need to include the `navigation` element in all the methods? The class already provides the context.

Reply:
Done in 5cbd5c5f10, bb219fe16b, 44fca51dea, 4c9c4708bd